### ISSUE TITLE
test

### ISSUE THREAD

**Issue by foxquan** [zzz0326/EPSNet#1]
Hello, I would like to use some other data to test. The data size of the validation set is unified to 2048*1024, but an error is reported when out=model(input).detach().numpy is executed.
error：进程已结束，退出代码137（interrupted by signal 9: SIGKILL）
